.eslintrc

```diff
@@ -25,7 +25,6 @@
     }
   },
   "rules": {
-    "@typescript-eslint/camelcase": "off",
```

Comment:
`CHANGELOG` here:

1. Alphabetize rules
1. Add new rule for PR, `@typescript-eslint/consistent-type-definitions": ["error", "interface"]`. 
    - `eslint --fix` automatically converts `type` definitions to `interface`. Most of this PR was automated 🙂 . 
1. Removes `.js` rules for files that no longer exist.